tests/scripts/helm.sh

```diff
@@ -26,19 +24,29 @@ detectArch() {
     esac
 }
 
-install() {
-    if ! helm_loc="$(type -p "helm")" || [[ -z ${helm_loc} ]]; then
-        # Download and unpack helm
-        local dist
-        dist="$(uname -s)"
+# Set the global HELM variable to the helm command. Install helm to a temp 
+# location if it's not present.
+install_helm() {
+    HELM="helm"
```

Comment:
I would be for moving the `HELM` var outside this function as was previously.

Reply:
ok. I agree it's cleaner.